### PR TITLE
Works section

### PR DESCRIPTION
**Pull request by emmanuelcortes2**
Linters setup & install

### CODE REVIEW

**Comment by emmanuelcortes2** [comment]
### My personal portfolio

- Added & styled works section
- Added readme file
- Added index.html
- Added style.css

File: style.css [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+.p-button:hover { background-color: #7f8cff; }
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  padding-top: 10px;
+  padding-bottom: 25px;
+  box-shadow: 2px 2px 2px #888;
+}
```
Comment (Abenezer-Tilahun) [inline style.css:188]: - [ ] Kindly, adjust "About Myself "section to have the the margins (top, bottom, left, right) defined in the design. 

Your project Look like this : 
Captur12

The Figma design Look likes this :

File: index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                    <ul>
+                      <li class="js-btn"><a href="#">JavaScript</a></li>
+                      <li class="ht-btn"><a href="#">Html</a></li>
+                      <li class="css-btn"><a href="#">CSS</a></li>
+                    </ul>
+                  </li>
+                  <li><a href="#">Frameworks</a></li>
+                  <li><a href="#">Skills</a></li>
+                </ul>
+                </nav>
```
Comment (Abenezer-Tilahun) [inline index.html:177]: - [ ] Kindly, try to Follow the Figma design, In your project for "Languages", "Frameworks","Skills" there is no arrow indication just like design template . You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
Your Project Look likes this : 
Capture1

The Figma design Look likes : 

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @emmanuelcortes2  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README 

almost

## Required Changes ♻️
_Check the comments below the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (emmanuelcortes2) [reply style.css:188]: Ok, problem solved

File: index.html (hunk 2 above)
Comment (emmanuelcortes2) [reply index.html:177]: Ok. Fixed the problem, added bottom border and dropdown icons

File: index.html [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="bottom-list">
+                    <li><button class="language-buttons" type="button">html</button></li>
+                    <li><button class="language-buttons" type="button">css</button></li>
+                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                </ul>
+                <button class="p-button" type="button">See project</button>
+            </div>
+            <div class="flex-card">
```
Comment (Kamba56) [inline index.html:111]: - [Optional] Please consider using an indentation of 2 spaces for readability in all your documents. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: style.css [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+  height: auto;
+}
+
+.flex-card {
+  width: 100%;
+  flex-direction: row;
+  border-radius: 25px;
+  padding-bottom: 20px;
+  padding-top: 50px;
+}
```
Comment (Kamba56) [inline style.css:102]: - [Optional] Please kindly consider adding a border to your work cards to match the design template. Also, add a background color to the work section to match the template,
## Your design
m1
## Figma template
ccc

File: style.css [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+  line-height: 24px;
+  background-color: transparent;
+  border: 2px solid #838de9;
+  padding: 15px;
+  margin: 4px 2px;
+  border-radius: 12px;
+}
+
+.p-button:hover { background-color: #7f8cff; }
+
```
Comment (Kamba56) [inline style.css:181]: - [Optional] Consider adding a hover and pressed state to all the buttons to match the Figma template.
Screenshot 2022-08-20 195928

**Review by Kamba56 — APPROVED** [review]
Hi @emmanuelcortes2, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
JimCarreyYesSirGIF

## Highlights
- [x] Correct use of Github flow
- [x] All linters pased
- [ ] Descriptive PR title and description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**